clients/da-vinci-client/src/main/java/com/linkedin/davinci/DaVinciBackend.java

```diff
@@ -247,6 +250,19 @@ public DaVinciBackend(
       cacheBackend = cacheConfig
           .map(objectCacheConfig -> new ObjectCacheBackend(clientConfig, objectCacheConfig, schemaRepository));
 
+      String storeName = clientConfig.getStoreName();
+
+      if (getRecordTransformer != null && getRecordTransformer.apply(0) != null) {
```

Comment:
What is the meaning of doing apply(0) here?  This seems kind of sketchy?

Reply:
There was a bug in my code that would cause `getRecordTransformer.apply(n)` to return null in some cases. I've addressed it and that check is no longer needed.